docs/developers/contributing/documentation/index.md

```diff
@@ -484,7 +506,9 @@ not execute code cells. `slimfast` will run the build in parallel, which can be
 significantly faster on multi-core machines (under a minute), while `slimfast-live`
 will open a browser preview and auto-rebuild any pages you edit. 
 See [the `-live` builds note](live-builds).
+````
 
+````{dropdown} napari gallery of examples only
```

Comment:
this one isn't strictly true, the docs will be built, just no notebook cells run. Not sure how best to communicate that.
Based on the first two, I I guess I suggest: 
```suggestion
````{dropdown} napari/docs and napari gallery of examples
```